src/openapi.yaml

```diff
@@ -47,7 +47,7 @@ info:
   license:
     name: EUPL 1.2
     url: https://opensource.org/licenses/EUPL-1.2
-  version: 1.1.0-alpha
+  version: 1.2.0-alpha
```

Comment:
als 1.1.0 nog in alpha is, dan is er toch geen noodzaak om het versienummer al te verhogen?